package.json

```diff
@@ -126,17 +132,28 @@
   },
   "jest": {
     "collectCoverage": true,
+    "clearMocks": true,
     "coverageReporters": [
       "lcov"
     ],
     "modulePathIgnorePatterns": [
-      "<rootDir>/packages/*.*/dist/*.*"
+      "<rootDir>/packages/*.*/dist/*.*",
+      "<rootDir>/packages/*.*/public/*.*",
+      "<rootDir>/packages/*.*/.cache/*.*"
     ],
     "setupFiles": [
       "./test.env.js"
     ],
     "snapshotSerializers": [
-      "enzyme-to-json/serializer"
+      "enzyme-to-json/serializer",
+      "<rootDir>/packages/next/styles/snapshot-serializer"
+    ],
+    "transform": {
+      "^.+\\.jsx?$": "babel-jest",
+      "\\.(css)$": "<rootDir>/packages/next/styles/jest-transform.js"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!@patternfly)"
     ],
     "roots": [
```

Comment:
would you mind adding : 
```
    "coveragePathIgnorePatterns": [
      "<rootDir>/packages/*.*/dist/*.*",
      "<rootDir>/packages/*.*/public/*.*",
      "<rootDir>/packages/*.*/.cache/*.*"
    ],
``` 
?

I think that may fix a current issue with our test reporting/coverage reports. It seems to correctly ignore `dist` folders when this is added.